### PR TITLE
add: `intellij-idea-community-bin`

### PR DESCRIPTION
**Pull request by Rockstar9604**

### CODE REVIEW

**Review by Elsie19 — CHANGES_REQUESTED** [review]
Run `shfmt` as described in [the setup](https://github.com/pacstall/pacstall-programs#how-to-setup-the-environment-for-pacscript-development) and in [pre-commit](https://github.com/pacstall/pacstall-programs/blob/15e832e3164c7e1999d1474bf96394cd4f021932/.pre-commit-config.yaml#L27).

Packages will not be merged unless they are run thru `pre-commit`, so make sure to do that!

---

<sub>This is a saved reply I have set up, so don't worry, I don't sound this generic IRL 😆</sub>

**Comment by Rockstar9604** [comment]
> Run `shfmt` as described in [the setup](https://github.com/pacstall/pacstall-programs#how-to-setup-the-environment-for-pacscript-development) and in [pre-commit](https://github.com/pacstall/pacstall-programs/blob/15e832e3164c7e1999d1474bf96394cd4f021932/.pre-commit-config.yaml#L27).
> 
> Packages will not be merged unless they are run thru `pre-commit`, so make sure to do that!
> 
> This is a saved reply I have set up, so don't worry, I don't sound this generic IRL 😆

Sorry, I thought I did.

Is it okay now?

File: packages/intellij-idea-community-bin/intellij-idea-community-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+if [[ ${CARCH} == amd64 ]]; then
+  url="https://download-cdn.jetbrains.com/idea/ideaIC-${pkgver}.tar.gz"
+  hash="5e376677d4b9810f4171700ce9d9d3ece58f245ce8d9f2011b05966df5dd7ddd"
+else
+  url="https://download-cdn.jetbrains.com/idea/ideaIC-${pkgver}-aarch64.tar.gz"
+  hash="cfc98c4a09f6740ef47a3dbc8f2e71a66a8aab2eae3fb1b901c2db5dcce5f060"
+fi
+repology=("project: intellij-idea-community")
+
+desktop_file_sum="20c489d224f83cf4b29174b8c398e256b1df0e9cb0b920890f39c5796404b293"
```
Comment (Elsie19) [inline packages/intellij-idea-community-bin/intellij-idea-community-bin.pacscript:18]:
```suggestion
desktop_file_sum="20c489d224f83cf4b29174b8c398e256b1df0e9cb0b920890f39c5796404b293"

```

**Review by Elsie19 — APPROVED** [review]

**Review by Elsie19 — CHANGES_REQUESTED** [review]
~~Why does GitHub require review text~~

File: packages/intellij-idea-community-bin/intellij-idea-community-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  hash="5e376677d4b9810f4171700ce9d9d3ece58f245ce8d9f2011b05966df5dd7ddd"
+else
+  url="https://download-cdn.jetbrains.com/idea/ideaIC-${pkgver}-aarch64.tar.gz"
+  hash="cfc98c4a09f6740ef47a3dbc8f2e71a66a8aab2eae3fb1b901c2db5dcce5f060"
+fi
+repology=("project: intellij-idea-community")
+desktop_file_sum="20c489d224f83cf4b29174b8c398e256b1df0e9cb0b920890f39c5796404b293"
+
+prepare() {
+  wget -qO "intellij-idea-community.desktop" "https://raw.githubusercontent.com/Rockstar9604/pacstall-programs/master/packages/${name}/intellij-idea-community.desktop"
```
Comment (Elsie19) [inline packages/intellij-idea-community-bin/intellij-idea-community-bin.pacscript:20]: change this to our repo (even though the path doesn't exist yet) and I can merge it then

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/intellij-idea-community-bin/intellij-idea-community-bin.pacscript (hunk 2 above)
Comment (Rockstar9604) [reply packages/intellij-idea-community-bin/intellij-idea-community-bin.pacscript]: You're right, sorry, I forgot to reciprocate the repo. I've fixed it.

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**